include/SFML/Config.hpp.in

```diff
@@ -35,10 +35,12 @@
 ////////////////////////////////////////////////////////////
 // Define the SFML version
 ////////////////////////////////////////////////////////////
-#define SFML_VERSION_MAJOR      3
-#define SFML_VERSION_MINOR      0
-#define SFML_VERSION_PATCH      0
-#define SFML_VERSION_IS_RELEASE false
+// clang-format off
+#define SFML_VERSION_MAJOR @SFML_VERSION_MAJOR@
+#define SFML_VERSION_MINOR @SFML_VERSION_MINOR@
+#define SFML_VERSION_PATCH @SFML_VERSION_PATCH@
+// clang-format on
+#cmakedefine01 SFML_VERSION_IS_RELEASE
```

Comment:
Why change this from a boolean to an integer? Is it feasible to keep this as a boolean literal?

Reply:
With a little bit of ceremony, yes.